OpenDreamShared/Compiler/Location.cs

```diff
@@ -4,7 +4,14 @@ namespace OpenDreamShared.Compiler
 {
     public readonly struct Location
     {
+        /// <summary>
+        /// For when DM location information can't be determined.
+        /// </summary>
         public static readonly Location Unknown = new Location();
+        /// <summary>
+        /// For when internal OpenDream warnings/errors are raised.
```

Comment:
This comment should be extended to include anything internal that needs a location, instead of just warnings/errors. An example of this would be the assignment expressions used in init procs, which currently use `Location.Unknown`.